README.md

```diff
@@ -1,44 +1,22 @@
-# integration-tests
-This project contains integration tests written in [cypress](https://www.cypress.io/) that will check the various parts of the Cacophony ecosystem work together.  These are intended to be run on both test and production, as well as developers computers.
+# Cypress api tests
```

Comment:
Opps we need to combine this as we probably need to retain some of this information.